client/js/components/ListEvent/ListEvent.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {View, TouchableOpacity, Image, Text} from 'react-native';
+//import AddToCalendar from '../CalendarButton';
```

Comment:
remove commented code if not needed.

Reply:
This can be uncommented once other branch is merged